src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+
+  transition: transform 300ms ease;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  box-sizing: border-box;
+
+  @include flex(center, space-between);
+
+  flex-direction: column;
+
+  &__image {
+    padding-inline: 3px;
+
+    & img {
+      width: 100%;
+      height: 134px;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    font-family: inherit;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin: 4px 0 16px;
+    font-family: inherit;
+    font-size: 10px;
+    line-height: 14px;
+    color: $other-color;
+  }
+
+  &__reviews {
+    @include flex(center, space-between);
+  }
+
+  &__count {
+    font-family: inherit;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    align-self: end;
+    color: $main-text-color;
+  }
+
+  &__price {
+    margin: 24px 0 16px;
+
+    @include flex(center, space-between);
+  }
+
+  &__label {
+    font-family: inherit;
+    font-size: 12px;
+    line-height: 18px;
+    color: $other-color;
+  }
+
+  &__value {
+    font-family: inherit;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__button {
+    height: 40px;
+    width: 100%;
+    border: none;
+    border-radius: 5px;
+    background-color: $color-button;
+    cursor: pointer;
+
+    font-family: inherit;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: rgba(255, 255, 255, 1);
+
+    &:hover {
+      transition:
+        color 300ms ease,
+        background-color 300ms ease;
```

Comment:
The transition property should be defined in a single line for better readability and to follow the code style rules. Consider combining these lines into one.